libs/@local/hash-graph-client/python/graph_client/models.py

```diff
@@ -342,30 +335,72 @@ class Schema4(BaseModel):
     type: str
 
 
-class UpdateDataTypeRequest(BaseModel):
+class VersionedURL(RootModel):
     model_config = ConfigDict(
         populate_by_name=True,
     )
-    actor_id: RecordCreatedById = Field(..., alias="actorId")
-    schema_: Schema4 = Field(
-        ..., alias="schema", description="The contents of a Data Type update request"
+    root: AnyUrl = Field(
+        ...,
+        description="The versioned URL of a Block Protocol ontology type (the $id of the schema). It should be of the form `${baseUrl}v/${versionNumber}`",
+        max_length=2048,
+        title="Versioned URL",
     )
-    type_to_update: str = Field(..., alias="typeToUpdate")
 
 
-class VersionedUrl(RootModel):
+class DataType(BaseModel):
```

Comment:
I wonder if we can use downstream either the custom models I've written (removing this) or use this as base class for the custom model. I think they match?

Reply:
As this is already a `BaseModel` this should be fine I guess?